docs/en/writing-running-appium/android/activity-startup.md

```diff
@@ -0,0 +1,34 @@
+## How To Troubleshoot Activities Startup
+
+
+### Capabilities
+
+`The Activity class is a crucial component of an Android app, and the way activities are launched and put together is a fundamental part of the platform's application model. Unlike programming paradigms in which apps are launched with a main() method, the Android system initiates code in an Activity instance by invoking specific callback methods that correspond to specific stages of its lifecycle.` (c) [Android Developer Documentation](https://developer.android.com/guide/components/activities/intro-activities)
+
+Appium needs to know package and activity names in order to properly initialize the application under test. This information is expected to be provided in driver capabilities and consists of the following keys:
+
+- `appActivity`: The name of the main application activity
+- `appPackage`: The identifier of the application package
+- `appWaitActivity`: The name of the application activity to wait for/which starts the first
+- `appWaitPackage`: The id of the application package to wait for/which starts the first
+- `appWaitDuration`: The maximum duration to wait until the `appWaitActivity` is focused in milliseconds (20000 by default)
+
+All these capabilities are optional. If they are not set explicitly then Appium tries to auto detect them by reading their values from the APK manifest. Although, if the application under test is supposed to be already installed on the device (`noReset=true`) then at least `appActivity`and `appPackage` options are required to be set, since no package manifest is available in such case. If you don't set `appWaitPackage` and `appWaitActivity` explicitly then these are getting assigned to `appPackage`/`appActivity` values automatically. For more details check on the implementation of `packageAndLaunchActivityFromManifest` method in https://github.com/appium/appium-adb/blob/master/lib/tools/android-manifest.js
```

Comment:
"method in the [appium-adb](https://github.com/appium/appium-adb/blob/master/lib/tools/android-manifest.js) package"

Reply:
fixed